tools/module_dependencies.py

```diff
@@ -74,13 +68,10 @@ def _import_dependencies(module: str) -> set[str]:
     for mod in test_modules:
         try:
             importlib.import_module(mod)
-        except (ImportError, pytest.skip.Exception):
+        except pytest.skip.Exception:  # raised by `pytest.importorskip`
```

Comment:
Were you getting any regular `ImportError`s? I don't locally.

Reply:
Depends on which modules you have installed. I didn't have all test dependencies installed locally.